### PR TITLE
Created verification result report

### PR DESCRIPTION
**Pull request by rishubn**
Writes report file to Results/ folder for lwc verification flow. Failures also copy the failed testvectors.txt file for the failed test

```
{
    "success": true,
    "TV:kats_for_verification": {
        "PDI": "/nhome/LWC_Benchmarking/unrestricted_2021/Xoodyak_GMU2/KAT/v1/kats_for_verification/pdi.txt",
        "SDI": "/nhome/LWC_Benchmarking/unrestricted_2021/Xoodyak_GMU2/KAT/v1/kats_for_verification/sdi.txt",
        "DO": "/nhome/LWC_Benchmarking/unrestricted_2021/Xoodyak_GMU2/KAT/v1/kats_for_verification/do.txt"
    },
    "TV:blanket_hash_test": {
        "postreset_cycles": "3714",
        "total_sim_time": "37270000 ps",
        "PDI": "/nhome/LWC_Benchmarking/unrestricted_2021/Xoodyak_GMU2/KAT/v1/blanket_hash_test/pdi.txt",
        "SDI": "/nhome/LWC_Benchmarking/unrestricted_2021/Xoodyak_GMU2/KAT/v1/blanket_hash_test/sdi.txt",
        "DO": "/nhome/LWC_Benchmarking/unrestricted_2021/Xoodyak_GMU2/KAT/v1/blanket_hash_test/do.txt"
    }
}
```

### CODE REVIEW

File: xeda/plugins/lwc/flows/timing_and_verification.py [hunk 1]
```diff
@@ -352,4 +353,11 @@ def parse_reports(self):
 
             results['TV:' + rc_name] = rc_results
 
-        results['success'] = True
+        
+        if results['success'] != False:
```
Comment (kammoh) [inline xeda/plugins/lwc/flows/timing_and_verification.py:357]: Why is this required?

**Review by kammoh — CHANGES_REQUESTED** [review]

File: xeda/plugins/lwc/flows/timing_and_verification.py (hunk 1 above)
Comment (rishubn) [reply xeda/plugins/lwc/flows/timing_and_verification.py:357]: I removed the return on line 340,  so `results['success']` may be set to false from a run. We do not want to overwrite that result, so we only set true if its not false.

**Review by kammoh — APPROVED** [review]
